Minichain_v0/main.py

```
@@ -0,0 +1,255 @@
+    address = key.verify_key.encode(encoder=HexEncoder).decode()
+    print(f"✅ New wallet created for port {port}")
+    print(f"📍 Your address: {address}")
+    return key
+
+
+def get_address(key: SigningKey) -> str:
+    """Get address from signing key."""
+    return key.verify_key.encode(encoder=HexEncoder).decode()
+
+
+def print_banner(port: int, address: str, is_bootstrap: bool):
```

Ruff (0.15.1)

[error] Boolean-typed positional argument in function definition

(FBT001)